### ISSUE TITLE
Error fetching proxmox_coordinator_disk_pmox01_/dev/sdb data

### ISSUE THREAD

**Issue by to4ko** [dougiteixeira/proxmoxve#257]
### Checklist

- [X] I have updated to the latest available Proxmox VE Custom Integration version, [see the latest version here](https://github.com/dougiteixeira/proxmoxve/releases/latest).

### Describe the issue you are experiencing

having below error on 2024.1.3-4-5 with integration v3.4 as well as with "main" version

debug logs uploaded with diagnistics info

```
Logger: custom_components.proxmoxve
Source: helpers/update_coordinator.py:332
Integration: Proxmox VE ([documentation](https://github.com/dougiteixeira/proxmoxve), [issues](https://github.com/dougiteixeira/proxmoxve/issues))
First occurred: 00:36:13 (1 occurrences)
Last logged: 00:36:13

Error fetching proxmox_coordinator_disk_pmox01_/dev/sdb data:
```

### In which version of Home Assistant Core do you have the problem?

core-2024.1.5

### What version of Proxmox VE Custom Integration has the issue?

3.4.0

### What version of Proxmox VE do you have the problem?

not sure but might be 4-5 releases ago

### Diagnostics information

[proxmoxve-992cbef246a84580ecd4504cadee31c4-Disk pmox01__dev_sda-3f2194ff16553fcfcc0b16a739a28326.json.txt](https://github.com/dougiteixeira/proxmoxve/files/14000504/proxmoxve-992cbef246a84580ecd4504cadee31c4-Disk.pmox01__dev_sda-3f2194ff16553fcfcc0b16a739a28326.json.txt)

### Additional information

[home-assistant_proxmoxve_2024-01-21T07-56-22.444Z.log](https://github.com/dougiteixeira/proxmoxve/files/14000505/home-assistant_proxmoxve_2024-01-21T07-56-22.444Z.log)

**Comment by dougiteixeira** [comment]
I don't see any problems with the disk entities, can you indicate if they are normal?

I would like you to perform the steps below again:
  * [Enable debugging for integration](https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging)
  * Reload the integration configuration entry.
  * [Disable debugging and download the generated file](https://www.home-assistant.io/docs/configuration/troubleshooting/#disable-debug-logging-and-download-logs)

Send it here so you can get a clue about the problem.

**Comment by to4ko** [comment]
attached

![image](https://github.com/dougiteixeira/proxmoxve/assets/45161980/9ab0f5e3-bc06-497a-a51e-d0ae83624a8d)



[home-assistant_proxmoxve_2024-01-21T13-20-49.525Z.log](https://github.com/dougiteixeira/proxmoxve/files/14001424/home-assistant_proxmoxve_2024-01-21T13-20-49.525Z.log)

**Comment by dougiteixeira** [comment]
Please test the main version and check if it works now.

**Comment by to4ko** [comment]
> Please test the main version and check if it works now.

doen. no error on latest 'main' version. thanks!

**Comment by dougiteixeira** [comment]
@to4ko, 
I believe there is another problem related to yours that must occur when the diagnostic file is generated.

Please, you can follow the steps below:
* [Enable integration debugging](https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging).
* [Download a diagnostic file](https://www.home-assistant.io/docs/configuration/troubleshooting/#download-diagnostics).
* [Disable integration debugging and download the generated log file](https://www.home-assistant.io/docs/configuration/troubleshooting/#disable-debug-logging-and-download-logs).
* Attach the diagnostic file and the generated log file here.

**Comment by to4ko** [comment]
> * Attach the diagnostic file and the generated log file here.

as requested 
[home-assistant_proxmoxve_2024-01-22T20-02-01.470Z.log](https://github.com/dougiteixeira/proxmoxve/files/14014841/home-assistant_proxmoxve_2024-01-22T20-02-01.470Z.log)
[config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt](https://github.com/dougiteixeira/proxmoxve/files/14014843/config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt)

**Comment by dougiteixeira** [comment]
> as requested
> [home-assistant_proxmoxve_2024-01-22T20-02-01.470Z.log](https://github.com/dougiteixeira/proxmoxve/files/14014841/home-assistant_proxmoxve_2024-01-22T20-02-01.470Z.log)
> [config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt](https://github.com/dougiteixeira/proxmoxve/files/14014843/config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt)

As I imagined... fixed with #263.

When you can, I would be very grateful if you would update to the main version, repeat the above process and send the files for validation.

**Comment by to4ko** [comment]
as requested #2 :)
[home-assistant_proxmoxve_2024-01-23T06-43-43.146Z.log](https://github.com/dougiteixeira/proxmoxve/files/14019380/home-assistant_proxmoxve_2024-01-23T06-43-43.146Z.log)
[config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt](https://github.com/dougiteixeira/proxmoxve/files/14019382/config_entry-proxmoxve-992cbef246a84580ecd4504cadee31c4.json.txt)
